crates/sui-json-rpc/src/read_api.rs

```diff
@@ -117,14 +117,38 @@ impl ReadApiServer for ReadApi {
     async fn get_objects_owned_by_address(
         &self,
         address: SuiAddress,
-    ) -> RpcResult<Vec<SuiObjectInfo>> {
-        Ok(self
+        // exclusive cursor if `Some`, otherwise start from the beginning
+        options: Option<SuiObjectDataOptions>,
+        cursor: Option<ObjectID>,
+        limit: Option<usize>,
+    ) -> RpcResult<ObjectsPage> {
```

Comment:
per discussion offline, let's add `at_checkpoint: Option<CheckpointId>,` to the end with docs comments